care/emr/resources/facility/spec.py

```
@@ -154,7 +154,10 @@ def perform_extra_deserialization(self, is_update, obj):
         obj.geo_organization = Organization.objects.filter(
             external_id=self.geo_organization, org_type="govt"
         ).first()
-        obj.facility_type = REVERSE_REVERSE_FACILITY_TYPES[self.facility_type]
+        try:
+            obj.facility_type = REVERSE_REVERSE_FACILITY_TYPES[self.facility_type]
+        except KeyError as exc:
+            raise ValueError(f"Invalid facility_type: {self.facility_type}") from exc
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)